modules/custom/d_update/src/Updater.php

```diff
@@ -264,4 +264,39 @@ public function installModules(array $modules, $enable_dependencies = TRUE) {
     return $this->moduleInstaller->install($modules, $enable_dependencies);
   }
 
+  /**
+   * Method creates new instance of existing blocks inside another theme.
+   *
+   * @param $subthemeName
+   *   Name of the subtheme to place block into.
+   *
+   * @param array $configs
+   *   List of blocks configs to instantiate.
+   */
+  public function instantiateBlocksForSubtheme($subthemeName, array $configs) {
+    foreach ($configs as $configName) {
+      $baseConfig = Drupal::config($configName)->getRawData();
```

Comment:
A gdybyśmy tak po prostu wzięli całą tablicę `$baseConfig` i podmienili w niej tylko `id` i `theme` to by zadziałało? Bo boję się, że pominiemy jakiś klucz w tej tabeli, który się rzadko pojawia